rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogoutResource.java

```diff
@@ -37,32 +37,44 @@
     @Context UriInfo uri;
 
     @Override
+    public Response redirectToLogout() {
+        URI dest = uri.getBaseUriBuilder().path(LogoutApi.class).build();
+
+        return Response.status(Status.OK)
+                .entity(String.format("<!DOCTYPE html>\n<body>\n" +
+                        "<script>\n" +
+                        "(function(c){var a=new window.XMLHttpRequest;" +
+                        // https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/open
```

Comment:
Could you also add a comment describing what the purpose of the two lines below? Inconvenient to have to piece it together from the Mozilla link. And (nitpick) is there any reason it needs to be so densely formatted?